app/views/layouts/_navigation.html.haml

```diff
@@ -7,13 +7,16 @@
       %ul.navbar-nav.me-auto.mb-2.mb-lg-0
         %li.nav-item
           - if user_signed_in?
-            %li= current_user.email
+            %li.nav-link= current_user.email
             %li= link_to 'Log out', destroy_user_session_path, method: :delete, class: 'nav-link'
+            %li= link_to 'Account', edit_user_registration_path, class: 'nav-link'
           - else
             %li
               = link_to 'Log In', new_user_session_path, class: 'nav-link'
             %li
               = link_to 'Sign Up', new_user_registration_path, class: 'nav-link'
         %li.nav-item.dropdown
         %li.nav-item
+          %li= link_to 'English', root_path(locale: :en)
+          %li= link_to 'Українська', root_path(locale: :ua)
```

Comment:
test it by byebug in application controller if it works and translations really work